### ISSUE TITLE
size of tensor doesn't match

### ISSUE THREAD

**Issue by LeonaCMZ** [NielsRogge/Transformers-Tutorials#83]
![image](https://user-images.githubusercontent.com/57559538/154808069-4829e6f2-7938-4e2b-890a-4fdf503e1828.png)

how to deal with it? Thanks!

**Comment by NielsRogge** [comment]
Hi,

I actually don't know which model or notebook you're talking about. So it'd be great if you can give some code to reproduce your error.

**Comment by LeonaCMZ** [comment]
It is for train model 
![image](https://user-images.githubusercontent.com/57559538/154827387-5efa5352-6b57-471d-a107-dd2b6f8245f8.png)
![image](https://user-images.githubusercontent.com/57559538/154827408-865481a8-11dc-4337-afe2-a80c9ff11245.png)

**Comment by JGillette71** [comment]
If I may add some context, I am facing the same issue. 
The file being referenced looks like _Custom Named Entity Recognition with BERT_. 
I am seeing the same error in line 35 of the training function. 
Error appears after running 

`for epoch in range(EPOCHS):
    print(f"Training epoch: {epoch + 1}")
    train(epoch)`

I made the updates from #74 

Both _active_accuracy_ and _flattened_predictions_ are tensors of size 128, so not sure why the error is showing up on this particular line. 

![error Capture](https://user-images.githubusercontent.com/70489145/156118176-1c45e876-b7d4-42ef-a251-f03339669197.PNG)

Thanks in advance for any assistance. If I find a solution, I will be sure to update this issue.

**Comment by oznurakyuz09** [comment]
Hello, 
I found the solution. 
When we change the model output from "loss, tr_logits" to "outputs", the value of tr_logits keeps the value from one of the previous cells.  Since tr_logits had the old value, it was causing the shape of active_logits and thus flattened_predictions to be incorrect.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/29836100/157518942-3e458b04-b4e0-4bc4-a09e-53247d1edcc8.png">

So I updated the code on my notebook as in the screenshoot below and it worked fine.

<img width="948" alt="image" src="https://user-images.githubusercontent.com/29836100/157520094-b60d2673-547a-4c5b-a248-73935cc2ac7a.png">
